### ISSUE TITLE
Broken link to or missing documentation

### ISSUE THREAD

**Issue by iburadempa** [fantix/aintq#1]
The link to https://aintq.readthedocs.io in README.rst is broken.

**Comment by fantix** [comment]
Right, thanks for checking! That's because we didn't build any documentation yet, I'll leave this issue open to track the documentation work.

**Comment by jokull** [comment]
Is the project dead? It’s a really nice approach to job scheduling. I do something similar with Redis and Flask here: https://github.com/TakumiHQ/doppler

**Comment by fantix** [comment]
It's on hold - I don't have time to work on it.
